eclipse-che/che-plugin-registry
# Disable digest generation

Signed-off-by: Eric Williams <ericwill@redhat.com>



### What does this PR do?
Disable digest generation on request. When building in a disconnect environment it may not always be possible to calculate the image digests.

### Screenshot/screencast of this PR

N/A

### What issues does this PR fix or reference?

Part of CRW-1648.

### How to test this PR?

Run `build.sh` with `--skip-digest-generation` and check the output. None of the containers will have digests calculated.

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)


### Reviewers

Reviewers, please comment how you tested the PR when approving it.
